airbyte-integrations/connectors/source-instagram/source_instagram/client/api.py

```diff
@@ -78,6 +79,22 @@ def list(self, fields: Sequence[str] = None) -> Iterator[dict]:
     def filter_input_fields(self, fields: Sequence[str] = None):
         return list(set(fields) - set(self.non_object_fields))
 
+    @backoff_policy
+    def load_next_page(self, instance: Cursor):
+        instance.load_next_page()
+
+    @backoff_policy
+    def get_instance_cursor(self, ig_user: IGUser, attribute: str, params: dict = None, fields: Sequence[str] = None) -> Cursor:
+        return getattr(ig_user, attribute)(params=params, fields=fields)
+
+    def pagination(self, instance: Cursor) -> Iterator[Any]:
+        yield from instance._queue
```

Comment:
It looks like we can't decorate pagination with backoff without accessing private attributes of the Cursor.
Let's document this approach via comment and insert TODO warning about possible breaking changes in Facebook API implementation.
We can consider creating an issue for facebook_business client.